allennlp/data/dataset_readers/interleaving_dataset_reader.py

```diff
@@ -52,6 +59,24 @@ def __init__(
             raise ConfigurationError(f"invalid scheme: {scheme}")
         self._scheme = scheme
 
+    @overrides
+    def _set_worker_info(self, info: Optional[WorkerInfo]) -> None:
+        """
+        Should only be used internally.
+        """
+        super()._set_worker_info(info)
+        for reader in self._readers.values():
+            reader._set_worker_info(info)
+
+    @overrides
+    def _set_distributed_info(self, info: Optional[DistributedInfo]) -> None:
+        """
+        Should only be used internally.
+        """
```

Comment:
I think if the method starts with an `_`, and there is nothing good to say in the docstring, just don't have the docstring. `_` methods don't go into the docs anyways, right?

Reply:
Yea, I think I accidentally copy and pasted this from the base class. Will remove.